cmd/compute-domain-kubelet-plugin/nvlib.go

```diff
@@ -236,31 +236,29 @@ func (l deviceLib) getDeviceMajor(name string) (int, error) {
 	foundCharDevices := false
 
 	for scanner.Scan() {
-		line := scanner.Text()
+		// remove any whitespace
```

Comment:
This removes (hopefully) just leading and trailing whitespace, not any whitespace (: